src/cudadev/Makefile

```diff
@@ -47,7 +47,8 @@ $(1)_CUSRC := $$(wildcard $(TARGET_DIR)/plugin-$(1)/*.cu)
 $(1)_OBJ := $$(patsubst $(SRC_DIR)%,$(OBJ_DIR)%,$$($(1)_SRC:%=%.o))
 $(1)_CUOBJ := $$(patsubst $(SRC_DIR)%,$(OBJ_DIR)%,$$($(1)_CUSRC:%=%.o))
 $(1)_DEP := $$($(1)_OBJ:$.o=$.d)
-ALL_DEPENDS += $$($(1)_DEP)
+$(1)_CUDEP := $$($(1)_CUOBJ:$.o=$.d)
+ALL_DEPENDS += $$($(1)_DEP) $$($(1)_CUDEP)
```

Comment:
Good catch! Could you separate the changes in the `Makefile` into a separate PR that could be integrated more quickly? This fix should be propagated to the `Makefile`s of the other programs as well.

Reply:
Now in https://github.com/cms-patatrack/pixeltrack-standalone/pull/217